pallets/asset-index/src/lib.rs

```diff
@@ -870,6 +870,17 @@ pub mod pallet {
             })
         }
 
+        fn chain_location(asset: &T::AssetId) -> Option<MultiLocation> {
+            let location = Self::native_asset_location(asset)?;
+            match (location.first(), location.at(1)) {
+                (Some(Junction::Parent), Some(Junction::Parachain(id))) => {
+                    Some((Junction::Parent, Junction::Parachain(*id)).into())
+                }
+                (Some(Junction::Parent), _) => Some(Junction::Parent.into()),
+                _ => None,
+            }
```

Comment:
This the same as `chain_part`? https://github.com/open-web3-stack/open-runtime-module-library/blob/55053241229a8e842dbb717c2bb281daa9fbdf04/traits/src/location.rs#L9 
can we use this directly on `location` instead?